segmentation/utils/spatial_analysis_test.py

```diff
@@ -287,3 +296,18 @@ def test_calculate_cluster_spatial_enrichment():
     assert stats.loc["Point8", "p_pos", "Pheno2", "Pheno1"] > .05
     assert stats.loc["Point8", "p_pos", "Pheno2", "Pheno1"] > .05
     assert abs(stats.loc["Point8", "z", "Pheno2", "Pheno1"]) < 2
+
+
+def test_create_neighborhood_matrix():
+    # get positive expression and distance matrices
+    all_data_pos = make_expression_matrix("positive")
+    dist_mat_pos = make_distance_matrix("positive")
+
+    counts, freqs = spatial_analysis.create_neighborhood_matrix(all_data_pos, dist_mat_pos, distlim=51)
+
+    # Test the counts values for both fovs
+    assert (counts.loc[:19, "Pheno3":"Pheno2"] == 8).all().all()
+    assert (counts.loc[:19, "Pheno1"] == 4).all()
+
+    assert (counts.loc[80:99, "Pheno1":"Pheno3"] == 8).all().all()
+    assert (counts.loc[80:99, "Pheno2"] == 4).all()
```

Comment:
Probably a total noob for asking this, but is there a reason we're not testing for negative and no enrichment matrices?